Makefile

```diff
@@ -35,4 +35,4 @@ test-no-warnings:
 	pytest --disable-warnings --cov=harmony tests
 
 cve-check:
-	safety check
+	safety check -i 51457
```

Comment:
Does this mean to ignore a vulnerability? If so is there a ticket to track fixing it?

Reply:
Just realized I can upgrade pytest which dropped the py (vulnerable) dependency. https://github.com/nasa/harmony-service-lib-py/pull/34/commits/44952435acb06e991229fb5495778777286d9cf5